sdk/textanalytics/Azure.AI.TextAnalytics/tests/TextAnalyticsClientLiveTests.cs

```diff
@@ -295,6 +296,71 @@ public async Task AnalyzeSentimentWithLanguageTest()
             Assert.AreEqual("Positive", docSentiment.Sentiment.ToString());
         }
 
+        [Test]
+        public async Task AnalyzeSentimentWithOpinionMining()
+        {
+            TextAnalyticsClient client = GetClient();
+            string document = "The park was clean and pretty. The bathrooms and restaurant were not clean.";
+
+            DocumentSentiment docSentiment = await client.AnalyzeSentimentAsync(document, options: new AnalyzeSentimentOptions() { IncludeOpinionMining = true });
+
+            CheckAnalyzeSentimentProperties(docSentiment, opinionMining:true);
+            Assert.AreEqual("Mixed", docSentiment.Sentiment.ToString());
+        }
+
+        [Test]
+        public async Task AnalyzeSentimentWithOpinionMiningEmpty()
+        {
+            TextAnalyticsClient client = GetClient();
+            string document = "That was the best day of my life!";
+
+            DocumentSentiment docSentiment = await client.AnalyzeSentimentAsync(document, "en", new AnalyzeSentimentOptions() { IncludeOpinionMining = true });
+
+            CheckAnalyzeSentimentProperties(docSentiment);
+            Assert.AreEqual("Positive", docSentiment.Sentiment.ToString());
+        }
+
+        [Test]
+        public async Task AnalyzeSentimentWithOpinionMiningNegated()
+        {
+            TextAnalyticsClient client = GetClient();
+            string document = "The bathrooms are not clean.";
+
+            DocumentSentiment docSentiment = await client.AnalyzeSentimentAsync(document, options: new AnalyzeSentimentOptions() { IncludeOpinionMining = true });
+
+            CheckAnalyzeSentimentProperties(docSentiment, opinionMining:true);
+            MinedOpinion minedOpinion = docSentiment.Sentences.FirstOrDefault().MinedOpinions.FirstOrDefault();
+            Assert.AreEqual("bathrooms", minedOpinion.Aspect.Text);
+            Assert.AreEqual(TextSentiment.Negative, minedOpinion.Aspect.Sentiment);
+            Assert.AreEqual("clean", minedOpinion.Opinions.FirstOrDefault().Text);
+            Assert.AreEqual(TextSentiment.Negative, minedOpinion.Opinions.FirstOrDefault().Sentiment);
+            Assert.IsTrue(minedOpinion.Opinions.FirstOrDefault().IsNegated);
+        }
+
+        [Test]
+        public async Task AnalyzeSentimentWithCancellationTest()
+        {
+            TextAnalyticsClient client = GetClient();
+            string document = "El mejor test del mundo!";
+
+            DocumentSentiment docSentiment = await client.AnalyzeSentimentAsync(document, cancellationToken: new System.Threading.CancellationToken());
```

Comment:
Just a small note here: if you're not using the cancellation token at all, you could just set it to `default`, or to `CancellationToken.None` if you want to be more explicit.